qa/acceptance/spec/lib/cli/logstash-plugin/install_spec.rb

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+require "fileutils"
+require "logstash/devutils/rake"
```

Comment:
I see you required it for the download version, might be we can do a finer grained requirement?